modules/agent-install-mirroring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,253 @@
+    - my-registry:443/ocp4-release/openshift
+    source: quay.io/openshift-release-dev/ocp-release
+  additionalTrustBundle: | <3>
+      -----BEGIN CERTIFICATE-----
+      <MY_TRUSTED_CA_CERT>
+      -----END CERTIFICATE-----
+----
+<1> A pull secret for the mirror registry.
+<2> Mirrors for sources and repositories for the release-image content. The certificate file can be an existing, trusted certificate authority,
+or the self-signed certificate that you generated for the mirror registry.
+<3> The certificate file used for the mirror registry.
+
+. Create the `agent-config.yaml` file:
++
+[source,yaml]
+----
+  cat > agent-config.yaml << EOF
+  apiVersion: v1alpha1
+  kind: AgentConfig
+  metadata:
+    name: sno-cluster
+  rendezvousIP: 192.168.111.80 <1>
+  hosts: <2>
+    - hostname: master-0
+      interfaces:
+        - name: eno1
+          macAddress: 00:ef:44:21:e6:a5
+      networkConfig:
+        interfaces:
+          - name: eno1
+            type: ethernet
+            state: up
+            mac-address: 00:ef:44:21:e6:a5
+            ipv4:
+              enabled: true
+              address:
+                - ip: 192.168.111.80
+                  prefix-length: 23
+              dhcp: false
+        dns-resolver:
+          config:
+            server:
+              - 192.168.111.1
+        routes:
+          config:
+            - destination: 0.0.0.0/0
+              next-hop-address: 192.168.111.2
+              next-hop-interface: eno1
+              table-id: 254
+  EOF
+----
++
+<1> This IP address is used to determine which node performs the bootstrapping process as well as running the `assisted-service` component.
+You must provide the IP address when you do not specify the node's IP addresses in the `networkConfig` parameter. If this address is not provided, one IP address is selected from the provided nodes's `networkConfig`.
+<2> The number of hosts defined must match the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters. When 3 master nodes and 0 worker nodes are defined in the `install-config.yaml` file,
+the number of hosts defined is 3. When 3 master nodes and 2 worker nodes are defined in the `install-config.yaml` file, the number of hosts defined is 5.
+
+. Mirror the images to a directory on the removable media:
++
+[source,terminal]
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
+
+. Upload the images to the local container registry from the removable media:
++
+[source,terminal]
+----
+$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror "file://openshift/release:${OCP_RELEASE}*" ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} <1>
+----
++
+<1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
```

Comment:
I think the alternate way of mirroring using the **oc mirror** command should be documented in a separate section

Reply:
I was referencing https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/index.html
In that, we have two methods 
oc adm
oc mirror

https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/installing-mirroring-installation-images.html  >> step v is the above command, is that wrong?